src/Esfa.Vacancy.UnitTests/CreateApprenticeship/Api/Mappings/GivenACreateApprenticeshipRequestMapper/WhenMappingFromApiParameters.cs

```diff
@@ -23,8 +23,8 @@ public void SetUp()
             var randomWageType = fixture.Create<int>();
```

Comment:
Just a question - How do you know when you use this randomise function that the random value returned is a valid value for the enum.